python/tvm/relay/frontend/tensorflow.py

```diff
@@ -639,7 +639,7 @@ def _impl(inputs, attr, params):
         end = size
         for i in range(data_dim):
             if size[i] == -1:
-                end[i] = data_shape[i] - begin[i]
+                end[i] = data_shape[i] - begin[i] + 1
```

Comment:
In tf doc, when size=-1. it means all remaining elements should be included. So here should be end[i] = data_shape[i]?

Reply:
Ok, I agree with you. It's changed now